src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertAdditionalServices.sql

```diff
@@ -187,6 +187,27 @@ BEGIN
 			    (@additionalServiceId, 1, 1, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', 1, 'GBP', @now, 599.99);
 	    END;
     END;
+    /***************************************************************************************************************************/
+    SET @solutionId = '100008-001';
```

Comment:
As per solution insert comment, the first part of the solution ID is the supplier I so this solution ID should be `'100007-002'`.